doc/Tutorials/Linked_Streams.ipynb

```diff
@@ -16,24 +16,24 @@
+    "In previous notebooks we discovered how the [DynamicMap](Dynamic_Map.ipynb) class allows us to declare objects in a lazy way to enable exploratory analysis of large parameter spaces. In the [Streams](Streams.ipynb) tutorial we learned how to interactively push updates to existing plots by declaring Streams on a DynamicMap. In this tutorial we will extend the idea to so called *linked* Streams, which allows complex interactions to be declared by specifying which events should be exposed when a plot is interacted with. By passing information about live interactions to a simple Python based callback, you will be able to build richer, even more interactive visualizations that enable seemless data exploration.\n",
     "\n",
-    "Some of the many possibilities this opens up include:\n",
+    "Some of the possibilities this opens up include:\n",
     "\n",
     "* Dynamically aggregating datasets of billions of datapoints depending on the plot axis ranges using the [datashader](https://anaconda.org/jbednar/holoviews_datashader/notebook) library.\n",
-    "* Responding to Tap and DoubleTap events to reveal more information in subplots.\n",
+    "* Responding to ``Tap`` and ``DoubleTap`` events to reveal more information in subplots.\n",
     "* Computing statistics in response to selections applied with box- and lasso-select tools.\n",
     "\n",
-    "Currently only the bokeh backend for HoloViews supports linked streams but in theory any backend can define callback which fire when a user zooms or pans or interacts with a plot.\n",
+    "Currently only the bokeh backend for HoloViews supports the linked streams system but the principles used should extend to any backend can define callbacks that fire when a user zooms or pans or interacts with a plot.\n",
     "\n",
     "<center><div class=\"alert alert-info\" role=\"alert\">To use and visualize <b>DynamicMap</b> or <b>Stream</b> objects you need to be running a live Jupyter server.<br>This tutorial assumes that it will be run in a live notebook environment.<br>\n",
     "When viewed statically, DynamicMaps will only show the first available Element,<br></div></center>\n",
     "\n",
     "## Available Linked Streams\n",
     "\n",
-    "There a huge number of ways one might want to interact with a plot. The HoloViews streams module aims to offer many of the most common interactions you might want to specify while making it easy to extend it with custom linked Streams. \n",
+    "There are a huge number of ways one might want to interact with a plot. The HoloViews streams module aims to expose many of the most common interactions you might want want to engage in, while also supporting extensibility via ustom linked Streams. \n",
```

Comment:
"engage in" seems awkward, maybe "utilize" or "employ"

Also "ustom linked Streams"